### ISSUE TITLE
Are there plans to migrate to Godot 4?

### ISSUE THREAD

**Issue by JuanFdS** [GDquest/godot-presentations#11]
I'll be preparing a talk about Godot, and i was thinking on using Godot 4 for the talk. This project seems really cool and I was thinking on using it for the talk.
Is there an alternative to this project, or a plan to migrate it? (I guess I won't be able to migrate it in time, but I think I'd use it in the future if it was available in Godot 4).

**Comment by NathanLovato** [comment]
No, there are no plans to migrate this at the moment. Some people used the presentation made with it but the project never quite got much interest otherwise.
